test/performance/load-test/main.go

```diff
@@ -81,6 +71,16 @@ func processResults(ctx context.Context, q *quickstore.Quickstore, results <-cha
 		serving.ServiceLabelKey: fmt.Sprintf("load-test-%s", *flavor),
 	})
 
+	stopDeploymentCh := make(chan struct{})
+	deploymentStatus := metrics.FetchDeploymentStatus(ctx, "default", selector, time.Second, stopDeploymentCh)
```

Comment:
I'd like to see if we can signal that these should stop by closing `deploymentStatus`

Reply:
I don't think we can do this, but I have another way to simplify this.